src/app/orcid/orcid.component.html

```diff
@@ -0,0 +1,17 @@
+<app-header>
+  <div fxLayout="row" fxLayoutGap="1rem">
+    <img src="../../../assets/svg/sub-nav/organization.svg" alt="organization icon" />
+    <span>ORCID</span>
+  </div>
+</app-header>
+
+<div fxLayout="column" fxLayoutAlign="start center">
+  <div fxLayout="row">
+    <mat-card>
+      Thank you for connecting your ORCID iD
+      <img src="../../assets/images/account-logos/orcid.svg" alt="ORCID iD logo" class="orcid-id-logo" />
+      <br />
+      You will now be automatically directed back to your Accounts page.
```

Comment:
Totally optional, but you could make the accounts page a link, in case the user doesn't feel like waiting; I've seen sites like that. Unless the guidelines don't allow it.